CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ Welcome to the Zulip community!
 The
 [Zulip community server](https://zulip.com/development-community/)
 is the primary communication forum for the Zulip community. It is a good
-place to start whether you have a question, as a new contributor, as a new
+place to start whether you have a question, are a new contributor, or a new
```

Comment:
Hey, @alya I have squashed the commit and added the screenshot here.

![contributing](https://user-images.githubusercontent.com/49565671/198346214-648f8ba0-9e79-4408-9155-09d0815c2e97.PNG)